src/method_ilan.jl

```diff
@@ -112,16 +114,16 @@ function ilan(
     a=Vector{T}(γ.^(0:2m+2)); a[1]=zero(T);
     local M0inv::LinSolver = create_linsolver(linsolvercreator,nep,σ)
     err=ones(m,m);
-    λ=zeros(T,m+1);
+    λ=zeros(T,5m+1);
```

Comment:
Why 5m? Is 5 arbitrary? Maybe it should be a solver parameter?

Reply:
It should be 2m+2, I will fix it.